Controllers/err.c.js

```diff
@@ -1,8 +1,7 @@
-exports.handleInvalidPaths = (req, res, next) => {
+exports.handleInvalidPaths = (err, req, res, next) => {
   res.status(404).send({ msg: "Invalid Path" });
   next(err);
```

Comment:
```js
  next(err);
```
No need to call next here. You have already sent the response and should only do one or the other.